reference_gen/node-docs.ts

```diff
@@ -84,13 +84,18 @@ for (const file of files) {
       )
     ) {
       if (
+        !exportable.wasForgotten() &&
         exportable.getKind() !== ts.SyntaxKind.ImportEqualsDeclaration &&
         !(exportable.getKind() === ts.SyntaxKind.ModuleDeclaration &&
           exportable.getName() === "global")
       ) {
-        exportable.setIsExported(
-          !EXCLUDE_MAP[fileName]?.includes(exportable.getName?.()),
-        );
+        if (exportable.getName?.() === "BuiltInModule") {
```

Comment:
this is an annoying workaround, but better for now to exclude it and re-introduce once problems related to it are fixed. causes several issues and have to increase max-heap to 6GB+